scripts/pin_the_slow_way.py

```diff
@@ -29,23 +29,23 @@
 
 
 pinned = {
-          'boost': 'boost 1.60.*',
+          'boost': 'boost 1.61.*',
           'bzip2': 'bzip2 1.0.*',
           'fontconfig': 'fontconfig 2.11.*',
-          'freetype': 'freetype 2.6*',
-          'hdf5': 'hdf5 1.8.17*',
+          'freetype': 'freetype 2.6.*',
+          'hdf5': 'hdf5 1.8.17',
           'icu': 'icu 56.*',
-          'jpeg': 'jpeg 9*',
-          'libnetcdf': 'libnetcdf 4.4*',
+          'jpeg': 'jpeg 9.*',
+          'libnetcdf': 'libnetcdf 4.4.*',
           'libpng': 'libpng >=1.6.21,<1.7',
           'libtiff': 'libtiff 4.0.*',
-          'ncurses': 'ncurses 5.9*',
-          'openssl': 'openssl 1.0*',
-          'readline': 'readline 6.2*',
+          'ncurses': 'ncurses 5.9.*',
+          'openssl': 'openssl 1.0.*',
+          'readline': 'readline 6.2.*',
```

Comment:
Similarly, `readline` should not have a `.` as they seem to use only 2 version numbers.